app/javascript/tests/patient/assessment/actions/MonitoringPeriod.test.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+import MonitoringPeriod from '../../../../components/patient/assessment/actions/MonitoringPeriod';
+import { mockPatient1, mockPatient2 } from '../../../mocks/mockPatients';
+
+import SymptomOnset from '../../../../components/patient/assessment/actions/SymptomOnset';
+import LastDateExposure from '../../../../components/patient/assessment/actions/LastDateExposure';
+import ExtendedIsolation from '../../../../components/patient/assessment/actions/ExtendedIsolation';
+
+function getWrapper(patient) {
+  return shallow(<MonitoringPeriod patient={patient} />);
+}
+
+describe('MonitoringPeriod', () => {
+  it('Properly renders symptom onset and extended isolation when patient is in isolation workflow', () => {
+    const wrapper = getWrapper(mockPatient1);
+    expect(wrapper.find(SymptomOnset).exists()).toBeTruthy();
+    expect(wrapper.find(LastDateExposure).exists()).toBeFalsy();
+    expect(wrapper.find(ExtendedIsolation).exists()).toBeTruthy();
+    expect(wrapper.find('#end_of_monitoring_date').exists()).toBeFalsy();
+  });
+
+  it('Properly renders symptom onset, last date of exposure, and end of monitoring when patient is in exposure workflow', () => {
+    const wrapper = getWrapper(mockPatient2);
+    expect(wrapper.find(SymptomOnset).exists()).toBeTruthy();
+    expect(wrapper.find(LastDateExposure).exists()).toBeTruthy();
+    expect(wrapper.find(ExtendedIsolation).exists()).toBeFalsy();
+    expect(wrapper.find('#end_of_monitoring_date').exists()).toBeTruthy();
```

Comment:
thanks for adding these! similar vein, can you check the rest of the rendering in this section is correct?

Reply:
https://github.com/SaraAlert/SaraAlert/pull/861/commits/a5f7d0708f903826cb1f4dec5169309b37c77dbb#diff-e15f1c6b962325b3f9d37b9159f62060066188401f072899685088ff987b792bR30